### PR TITLE
Feature/featured message

### PR DESCRIPTION
**Pull request by omeneses**
1. Link to Trello's task: https://trello.com/c/uog6WpbB/17-mobile-mensaje-secundario-3
2. task's design in zeplin: https://app.zeplin.io/project/5f15bfe784a1689e60712353/screen/5f28bee24633762fd180024d
3. Component's screenshot
Screen Shot 2020-08-07 at 10 39 06 PM
4. Code implemented using Grid.

### CODE REVIEW

**Review by juan-ortizc — APPROVED** [review]

File: src/js/components/pageTitle.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const pageTitle = data => {
```
Comment (jsebax) [inline src/js/components/pageTitle.js]: What is this file added in this PR? It was already develop in a previous one

Comment (jsebax) [reply src/js/components/pageTitle.js]: Please perform a `git pull origin develop` from your branch to have the latest changes

File: src/scss/abstracts/_variables.scss [hunk 2]
```diff
@@ -1,14 +1,13 @@
...
-$black: #000000;
+$black: #000;
 $blue-grey: #474f78;
 $blue-grey-dark: #323755;
 $blue-grey-light: #adb0c4;
 $green: #0c5a45;
 $green-dark: #043326;
 $grey: #d8d8d8;
-$orange: #ffcc36;
+$orange: #ffb000;
```
Comment (ArleyAV) [inline src/scss/abstracts/_variables.scss:11]: Why did you change the value of this color?

File: src/js/components/pageTitle.js (hunk 1 above)
Comment (omeneses) [reply src/js/components/pageTitle.js]: Done!!!

File: src/scss/abstracts/_variables.scss (hunk 2 above)
Comment (omeneses) [reply src/scss/abstracts/_variables.scss:11]: That is the right color for orange.  Please see the attached screenshot
Screen Shot 2020-08-17 at 9 02 11 PM

**Review by ArleyAV — APPROVED** [review]
Well done Omar

**Review by jsebax — APPROVED** [review]
